src/components/currencies/currenciesTop.js

```diff
@@ -0,0 +1,26 @@
+import Grid from '@mui/material/Grid';
+
+import getCurrenDateTime from '../../common/currentDateTime';
+// import stock from '../../images/stocks.svg';
```

Comment:
- [ ]  Kindly remove this commented code. This will make your codebase cleaner and more professional